docker/sidekiq.sh

```diff
@@ -1,8 +1,2 @@
 #!/bin/sh
-runuser -p app -c 'cd /home/app/decidim-diba && bundle exec sidekiq -d -L "log/sidekiq.log" -e production -C config/sidekiq.yml'
-
-# The following is simpler but sidekiq is executed as root and the files it creates are also created as root:root,
-# this causes permissions problems when passenger tries to access files in tmp/ that were generated by sidekiq processes,
-# mainly in /tmp/cache
-#
-# bundle exec sidekiq -d -L "log/sidekiq.log" -e production -C config/sidekiq.yml
```

Comment:
why are we changing this? it is a fix that we needed to do. Please, do not change anything in the file.